src/modules/vmount/input_mavlink.cpp

```diff
@@ -303,7 +303,7 @@ int InputMavlinkCmdMount::update_impl(unsigned int timeout_ms, ControlData **con
 						break;
 
 					case vehicle_command_s::VEHICLE_MOUNT_MODE_GPS_POINT:
-						control_data_set_lon_lat((double)vehicle_command.param2, (double)vehicle_command.param1, vehicle_command.param3);
+						control_data_set_lon_lat((double)vehicle_command.param6, (double)vehicle_command.param5, vehicle_command.param7);
```

Comment:
From https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL it should be:
```suggestion
						control_data_set_lon_lat((double)vehicle_command.param6, (double)vehicle_command.param5, vehicle_command.param4);
```

Reply:
You are right. Thanks!